src/base/bittorrent/session.cpp

```diff
@@ -155,6 +155,19 @@ namespace
 
         return expanded;
     }
+
+    QStringList findAllFiles(const QString &folderPath)
+    {
+        QStringList files;
```

Comment:
@evsh:  AKAIK, symbolic links are handled automatically by the filesystem / operating system.  I use them for launching a directory-locked Java program on my windows machine while storing its data on another drive, for example.  

I have, however, observed folder permissions conflicts with certain other programs on occasion.  I haven't tested with qBittorrent, however.